package.json

```diff
@@ -156,10 +157,11 @@
     "fetch-cookie": "^0.6.0",
     "fixed-data-table": "^0.6.4",
     "form-data": "^2.3.1",
-    "fs-extra": "https://registry.npmjs.org/fs-extra/-/fs-extra-0.26.7.tgz",
+    "fs-extra": "^4.0.2",
+    "immutable": "^3.8.2",
```

Comment:
I'm pretty sure that we've gotten rid of immutable everywhere, and that these prop type descriptions https://github.com/plotly/falcon-sql-client/blob/92755984257409dcdcfb29231e272d8e4d2d174d/app/components/Settings/Logger/Logger.react.js#L33 are actually incorrect. We can change this in a future PR though

Reply:
I kept it to silence the warning that `immutable` is a peer depedency of `react-immutable-proptypes`.

Reply:
right - i mean, we can probably remove all of the `react-immutable-proptypes` throughout our code as well